config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```diff
@@ -179,3 +179,48 @@ presets:
         path: ca.crt
       - key: vmc-vpn-tls.key
         path: tls.key
+- labels:
+    # For kubernetes-sigs/cluster-api-provider-vsphere janitor
+    preset-vsphere-janitor-config: "true"
+  env:
+  - name: BOSKOS_HOST
+    value: http://192.168.6.138:32222
+  - name: GOVC_URL
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-url
+  - name: GOVC_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-cluster-api-provider-vsphere-user
```

Comment:
Let's add a TODO here and on the next env var that this should be changed to the janitor user as soon as that user is available